assets/app/scripts/filters/resources.js

```diff
@@ -779,16 +779,8 @@ angular.module('openshiftConsole')
       }
     };
   })
-  .filter('humanizeKind', function () {
-    return function(kind) {
-      if (!kind) {
-        return kind;
-      }
-
-      // ReplicationController -> Replication Controller
-      // https://lodash.com/docs#startCase
-      return _.startCase(kind);
-    };
+  .filter('humanizeKind', function (startCaseFilter) {
```

Comment:
Wondering if we ever want or need to do something different someday, e.g. ReplicationController -> Deployment? Or a kind where startCase doesn't work?